### DISCUSSION TITLE
[Feature] <AI image feature people & geo be available to shared users>

### DISCUSSION THREAD

**Discussion by kalo84** [immich-app/immich#5448] [Feature Request]
### The feature

Allow shared users to be able to see AI generated image info like People, geo location and others.
Currently,  a shared user can only see the image and the phone that took it as well as who is sharing the photo.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by kladderadeng** [comment]
I think this is the same as this?
https://github.com/immich-app/immich/discussions/5457

I would love to see this too. Currently every family member needs to import the same libraries and tag all the people again to get the search and people feature working for them.
